packages/dds/tree/src/shared-tree-core/messageFormat.ts

```diff
@@ -52,13 +52,18 @@ export const MessageFormatVersion = strictEnum("MessageFormatVersion", {
 	 * Only used for testing shared branches.
 	 */
 	vSharedBranches: "shared-branches|v0.1",
+	/**
+	 * Not yet released.
+	 * Only used for version alpha constraints.
+	 */
+	vAlphaConstraints: "alphaconstraints|v0.1",
```

Comment:
The fewer blue squiggles the better
```suggestion
	vAlphaConstraints: "alpha-constraints|v0.1",
```